documentation/how-to/prod-checklist.md

```diff
@@ -0,0 +1,81 @@
+---
+title: {Production Readiness Checklist}
+---
+
+This guide gives
+{{ glossary_tooltip term_id="api-provider" text="API Providers" }} a quick
+reference of actions they need to complete before publishing their Gateway
+Services to the {{ glossary_tooltip term_id="api-directory" }}.
+
+## First time activities
+
+- Learn about and understand the benefits of the
```

Comment:
It might be nice the use the checkbox for each point. I believe it is `- []` but you can play around if you think that might be appropriate